debian/bin/build

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env bash
 
 electron-packager . \
-  --ignore=node_modules/electron-prebuilt \
-  --ignore=node_modules/electron-packager \
```

Comment:
Yes, we can remove that because the option `--prune true` will remove all the `devDependencies` listed in `package.json`. So, right now, we are ignoring `electron`, `electron-installer-debian`, `electron-packager`, `elecetron-rebuild` and `foreman`.
You can check the `electron-packager` API, specially the [ignore](https://github.com/electron-userland/electron-packager/blob/master/docs/api.md#ignore) and [prune](https://github.com/electron-userland/electron-packager/blob/master/docs/api.md#prune) options.

This code, together with the changes in `package.json` have actually reduced the size of the `.deb` by a couple of Mbs.
You should consider using similar options when building for other OSs, it will make the code cleaner too.